merchstore/models.py

```diff
@@ -9,24 +10,35 @@ class ProductType(models.Model):
 
     class Meta:
         ordering = ["name"]
-        verbose_name = "ProductType"
-        verbose_name_plural = "ProductTypes"
+        verbose_name = "Product Type"
+        verbose_name_plural = "Product Types"
 
     def __str__(self):
         return self.name
 
-    def get_absolute_url(self):
-        return reverse("merchstore_variety", kwargs={"product_type": self.name})
-
 
 class Product(models.Model):
-
     name = models.CharField(max_length=255)
     product_type = models.ForeignKey(
         ProductType, null=True, on_delete=models.SET_NULL, related_name="ProductType"
     )
-    price = models.DecimalField(max_digits=7, decimal_places=2)
+    owner = models.ForeignKey(
+        Profile, null=False, on_delete=models.CASCADE, related_name="Owner", default=""
+    )
     description = models.TextField(default="A buyable item of this merch store.")
+    price = models.DecimalField(max_digits=24, decimal_places=2)
+    stock = models.PositiveIntegerField(default=0)
+
+    class ProductStatusChoices(models.TextChoices):
```

Comment:
Move this to just below `class Product (models.Model):` aka this should go first.